Car_Localizer/Car_Localizer/Training/training_script.py

```diff
@@ -0,0 +1,142 @@
+import os
+os.environ['KMP_DUPLICATE_LIB_OK'] = 'True'
+import numpy as np
+import pandas as pd
+import seaborn as sns
+from tqdm.notebook import tqdm
+import matplotlib.pyplot as plt
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torchvision import transforms, datasets
+from torch.utils.data import DataLoader, SubsetRandomSampler
+from sklearn.metrics import confusion_matrix
+sns.set_style('darkgrid')
+from model_vehicle_classifier import VehicleClassifier
+from custom_functions import get_mean_and_std, binary_acc
+# from Model import Model_First
+np.random.seed(0)
+torch.manual_seed(0)
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+
+print("In our project we're using =>", device)
+root_dir = "/Users/sebastiankowalczykiewicz/Car_Localizer/Car_Localizer/Car_Localizer/Training"
```

Comment:
`root_dir` should not depend on your path, you local path instead according to python project

Reply:
Corrected